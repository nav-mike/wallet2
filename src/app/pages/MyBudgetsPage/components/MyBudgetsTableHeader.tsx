import React from 'react';

export const MyBudgetsTableHeader = () => {
  return (
    <thead>
      <tr>
        <td>Year</td>
        <td>Month</td>
        <td>Value</td>
      </tr>
    </thead>
  );
};
