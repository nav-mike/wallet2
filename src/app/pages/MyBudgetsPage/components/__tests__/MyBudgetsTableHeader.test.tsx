import { render } from '@testing-library/react';
import * as React from 'react';
import { MyBudgetsTableHeader } from '../MyBudgetsTableHeader';

describe('<MyBudgetsTableHeader />', () => {
  it('should render and match the snapshot', () => {
    const header = render(
      <table>
        <MyBudgetsTableHeader />
      </table>,
    );
    expect(header.container.firstChild).toMatchSnapshot();
  });
});
