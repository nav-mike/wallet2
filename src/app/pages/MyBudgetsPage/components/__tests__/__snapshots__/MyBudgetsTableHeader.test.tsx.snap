// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MyBudgetsTableHeader /> should render and match the snapshot 1`] = `
<table>
  <thead>
    <tr>
      <td>
        Year
      </td>
      <td>
        Month
      </td>
      <td>
        Value
      </td>
    </tr>
  </thead>
</table>
`;
